ops/cmd/apply_hardforks/main.go

```diff
@@ -67,10 +66,14 @@ func processSuperchainDir(wd string, superchain config.Superchain) error {
 
 	for _, cfg := range cfgs {
 		chainCfg := cfg.Config
-		if chainCfg.SuperchainTime != nil && int64(*chainCfg.SuperchainTime) < time.Now().Unix() {
```

Comment:
The nil check on `SuperchainTime` now happens _inside_ `config.CopyHardforks` (makes it easier to unit test). 

The comparison between `SuperchainTime` and `time.Now()` has been removed. Hardfork timestamps are _static_ configuration and should not depend on the time when the configuration was generated (only on the relative timestamps of the superchain-wide hardforks, individual chain genesis times, superchain_time and individual hardfork overrides). See unit tests for examples.